packages/fx-core/src/plugins/resource/function/utils/azure-client.ts

```diff
@@ -70,6 +83,30 @@ export class AzureLib {
     return _t;
   }
 
+  public static async findResourceProvider(
+    client: Providers,
+    namespace: string
+  ): Promise<Provider | undefined> {
+    const provider = await client.get(namespace);
```

Comment:
Question: is it possible that user has no access to ResourceProvider?

Reply:
Good question. I didn't find any permission named `**/ResourceProvider/read` and provision succeed even I limit my permissions to only ResourceGroup/Read&write, Web/* and Storage/*. I assume user can always get resource provider.